feat(changelog): kubernetes-changed-new-size-limits-for-etcd-stores-in-k 2024-09-03 (#3668)
scaleway/docs-content: changelog/september2024/2024-09-03-kubernetes-changed-new-size-limits-for-etcd-stores-in-k.mdx

---

> title: New Size Limits for ETCD Stores in Kubernetes Control Planes

Reviewer: ```suggestion
title: New size limits for ETCD stores in Kubernetes control planes
```

status: changed
author:

>     fullname: ''

Reviewer: ```suggestion
    fullname: 'Join the #k8s channel on Slack.'
```

    url: 'https://slack.scaleway.com'
date: 2024-09-03
category: containers
product: kubernetes
---

> To maintain optimal performance of your Scaleway managed Kubernetes clusters, etcd stores in control planes are now subject to size limits: 55MB for mutualized control planes and 200MB for dedicated control planes.

Reviewer: ```suggestion
To maintain optimal performance of your managed Scaleway Kubernetes clusters, etcd stores in control planes are now subject to size limits: 55MB for mutualized control planes and 200MB for dedicated control planes.
```

> To maintain optimal performance of your managed Scaleway Kubernetes clusters, etcd stores in control planes are now subject to size limits: 55MB for mutualized control planes and 200MB for dedicated control planes.

Reviewer 2: ```suggestion
To maintain optimal performance of your managed Scaleway Kubernetes clusters, etcd stores in control planes are now subject to size limits: 55 MB for mutualized control planes and 200 MB for dedicated control planes.
```

> For more details, please refer to our [dedicated documentation](https://www.scaleway.com/en/docs/containers/kubernetes/reference-content/kubernetes-control-plane-offers/).

Reviewer: ```suggestion
For more details, please refer to our [dedicated documentation](/containers/kubernetes/reference-content/kubernetes-control-plane-offers/).
```
